lib/msal-common/src/uCache/serialize/CacheInterface.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Deserializer } from "./Deserializer";
+import { AccessTokenCache, IdTokenCache, RefreshTokenCache, AccountCache, AppMetadataCache, StringDict } from "../../utils/MsalTypes";
+
+export type JSONContent = {
+    accessTokens: StringDict;
+    idTokens: StringDict;
+    refreshTokens: StringDict;
+    accounts: StringDict;
+    appMetadata: StringDict;
+};
+
+export class CacheInterface {
+
+    /**
+     * serialize the JSON blob
+     * @param data
+     */
+    static serializeJSONBlob(data: string): string {
+        return JSON.stringify(data);
+    }
+
+    /**
+     * Parse the JSON blob in memory and deserialize the content
+     * @param cachedJson
+     */
+    static deserializeJSONBlob(jsonFile?: any): JSONContent {
+        let jsonContent: JSONContent = {
+            accessTokens: {},
+            idTokens: {},
+            refreshTokens: {},
+            accounts: {},
+            appMetadata: {}
+        };
+
+        const ParsedJSON = jsonFile ? jsonFile : null;
+        if (!ParsedJSON) {
+            return jsonContent;
+        }
```

Comment:
```js
if (!jsonFile) {
  return jsonContent;
}
```

Reply:
Also, this seems weird to return that default object?

Reply:
Not sure I see the value of that, and I would prefer to not return default-style objects (which are somewhat meaningless, in my opinion).  e.g. returning `null`

Reply:
This is mostly glue code to append to in memory list, I had the same objection. probably a better idea is to invoke this API only if the json is read from external memory. Will do the needed changes.

Reply:
Done